src/gatsby-node.js

```diff
@@ -20,6 +19,23 @@ const logMessage = (message, showLogs) => {
   }
 };
 
+// Helper to process an object of params
+// Converts to a string that can be added to a query
+// Query format: '?key=value'
+const processParams = params =>
+  params
+    ? '?' +
+      Object.entries(params)
+        .map(([key, value]) => {
+          if (Array.isArray(value)) {
+            return value.map(valueItem => `${key}=${valueItem}`).join();
+          } else {
+            return `${key}=${value}`;
+          }
+        })
+        .join()
+    : '';
+
```

Comment:
After the first argument, the rest must be separated with &

url?arg=value&arg=value

Reply:
Oh, arguments must be uriencoded.

Reply:
Oh damn, I'm so sorry, I should've done proper research and checked the code before pushing. This is my bad!